draft-packages/rating-scale-legacy/KaizenDraft/RatingScaleLegacy/RatingScaleLegacy.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { useState, createRef } from "react"
+
+import classNames from "classnames"
+import { Paragraph } from "@kaizen/component-library"
+import determineSelectionFromKeyPress from "./helpers/determineSelectionFromKeyPress"
+import { Scale, ScaleItem, ScaleValueResponse } from "./types"
+
+const styles = require("./styles.module.scss")
+
+type ItemRefs = Array<{
+  value: ScaleValueResponse
+  ref: { current: null | HTMLDivElement }
+}>
+
+export interface RatingScaleProps {
+  labelId: string
+  scale: Scale
+  selectedItem: ScaleItem | null
+  questionId: string
```

Comment:
This `questionId` prop doesn't make sense for the Kaizen API. It's only used to make automation ids, so I'd recommend changing this to an `automationId` prop, which is placed on the container div, and the two places that are using this prop can be updated to be something like 
```
data-automation-id={`${props.automationId}-legend`}
data-automation-id={`${props.automationId}-item-${item.value}`}
```